### ISSUE TITLE
HRR csv file contains seven columns

### ISSUE THREAD

**Issue by gforney** [firemodels/fds#211]
```
Please complete the following lines...

Application Version:
SVN Revision Number:
Compile Date:
Operating System:XP

Describe details of the issue below:
One of my HRR.csv files contains seven columns. The user's guide (12.3.1)
details six columns. The last column of the first two file records are
ZONE_01 for record 1 and atm for record 2. 

```

Original issue reported on code.google.com by `ferrante@firehousemail.com` on 2007-11-26 02:29:42

**Comment by gforney** [comment]
```
This is the pressure, in atmospheres, of ZONE_01, which I assume means that you have

not specified any OPEN boundaries to the outside and thus your entire domain is a 
sealed compartment. Or, you have specified a pressure ZONE in your input file, and

this is the pressure of that zone. 

This is not a bug, but could you clarify which of the above possibilities pertains

to you.
```

Original issue reported on code.google.com by `mcgratta` on 2007-11-26 20:30:03
- **Labels added**: OpSys-All

**Comment by gforney** [comment]
```
I've specified a ZONE which encompasses the entire mesh.

&MESH IJK=104,42,36 XB=0.0,14.0,0.0,7.0,0.0,5.5 /
&ZONE XB=0.0,14.0,0.0,7.0,0.0,5.5 /
```

Original issue reported on code.google.com by `ferrante@firehousemail.com` on 2007-11-26 21:44:31

**Comment by gforney** [comment]
```
OK -- during the beta testing of FDS 5, we required explicit input of the ZONE if 
there was no opening to the outside. It is now done automatically, as we found it a

bit of a pain to have to specify something that FDS could easily figure out. So 
everything appears to be working as designed. I will add a section on the format of

the _hrr.csv file to the User's Guide.
```

Original issue reported on code.google.com by `mcgratta` on 2007-11-26 22:27:56

**Comment by gforney** [comment]
```
This should be included in 5.0.3 and the online User Guide.
```

Original issue reported on code.google.com by `mcgratta` on 2007-12-10 23:05:37
